jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -553,15 +553,16 @@ def run(self):
         )
 
         # Button with the onclick event to swap embedded notebook back to examples.
+        # This includes a 'hidden' class by default to hide until the page is loaded.
         go_back_button_html = (
-            '<button class="try_examples_button" '
+            '<button class="try_examples_button hidden" '
```

Comment:
From what I can tell, these buttons are already within a `.hidden` container.
```suggestion
            '<button class="try_examples_button" '
```